### ISSUE TITLE
Run this program completely

### ISSUE THREAD

**Issue by luoruijie** [dangviethieu/fbcrawl#1]
![153ef5854d2654f37a308f68377f9f9](https://user-images.githubusercontent.com/50574451/69481812-8385a300-0e4f-11ea-8e9e-9b4059575653.png)

After I run the fbcrawl-master, I follow the process to log in to FireFox http://127.0.0.1:5004/, but when I click the login in button in the Login interface, it does not jump, the following figure Is it a display on my Pycharm, can I leave a contact? Can be Facebook or Wechat, I can also sponsor
Wechat: luoruijie1996
Facebook: Soikit LO

![21e6851335f987d2091bc38bddaf451](https://user-images.githubusercontent.com/50574451/69481789-4b7e6000-0e4f-11ea-8a26-6dc9d5a8b6cb.png)

**Comment by luoruijie** [comment]
![image](https://user-images.githubusercontent.com/50574451/69495276-1bdb6080-0f00-11ea-9f89-57c01ad6af4a.png)
Hello, I have already arrived at this interface. The last question is solved, but I will not run these interfaces. To be precise, I don't know how to start. Can you give me a more detailed manual? ?

**Comment by dangviethieu** [comment]
@luoruijie Hi. Thanks for question!
I used **scrapy** for crawling, I checkout it from: https://github.com/rugantio/fbcrawl
please ask me if you have any question!

**Comment by luoruijie** [comment]
Now I need to know if I can store the data that I crawled down from Facebook into the database when I execute Python main.app and run the code in this picture on this interface

![image](https://user-images.githubusercontent.com/50574451/69708433-91307680-1136-11ea-92c3-7759317a95bb.png)

**Comment by luoruijie** [comment]
Of course, the parameters of the code in this picture will be modified.

**Comment by dangviethieu** [comment]
Yes, you can call this command in terminal
And then, modifying in fbcrawl/pipelines.py to store data crawled to DB
This is example: https://realpython.com/web-scraping-with-scrapy-and-mongodb/
